src/scipp/plotting/formatters.py

```diff
@@ -225,6 +225,7 @@ def make_formatter(array, key, dim):
             formatter["custom_locator"] = True
         elif kind == Kind.datetime:
             coord = _get_or_make_coord(array, dim)
+            coord = coord - coord.min()
```

Comment:
I feel (also when I made the changes in #2446) a lot of this is duplicated between `formatter.py` and `model.py`.